packages/vda-sbt-client/src/blockchain/blockchainApi.ts

```diff
@@ -0,0 +1,215 @@
+import {
+    VeridaSelfTransactionConfig,
+    VeridaMetaTransactionConfig,
+    CallType,
+    getVeridaContract,
+    VeridaContract
+} from "@verida/web3"
+import { ethers } from "ethers";
+import { getContractInfoForNetwork } from "./config";
+import { getVeridaSignWithNonce, interpretIdentifier } from "./helpers";
+
+/**
+ * Interface for vda-name-client instance creation. Same as VDA-DID configuration
+ * @param identifier: DID
+ * @param signKey: private key of DID. Used to generate signature in transactions to chains
+ * @param chainNameOrId: Target chain name or chain id.
+ * @param callType : VDA-DID run mode. Values from vda-did-resolver
+ * @param web3Options: Web3 configuration depending on call type. Values from vda-did-resolver
+ */
+export interface SBTClientConfig {
+    identifier: string;
+    signKey: string;
+    chainNameOrId?: string | number;
+
+    callType: CallType;
+    web3Options: VeridaSelfTransactionConfig | VeridaMetaTransactionConfig;
+}
+
+export interface SBTInfo {
```

Comment:
@ITStar10 Add documentation that explains the purpose of each attribute

Reply:
Updated.